package.json

```diff
@@ -51,7 +52,9 @@
     "javascript"
   ],
   "author": "grtjn",
-  "contributors": ["pmcelwee"],
+  "contributors": [
+    "patrickmcelwee"
```

Comment:
I was using your npm handle, rather than your github one, but if you prefer your github one, that would be fine by me.. ;-)

Reply:
Thnx, including this in release 0.6.0..